src/main/java/com/googleinterns/zoomtube/servlets/DiscussionServlet.java

```diff
@@ -63,24 +63,45 @@ public void init() throws ServletException {
 
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    if (request.getParameter(PARAM_LECTURE) == null) {
+      response.sendError(HttpServletResponse.SC_BAD_REQUEST, "Missing lecture parameter.");
```

Comment:
nit: can you create constants for the error messages?

Reply:
Done!